libs/api-bundle/src/RequestMapper/ArgumentResolver.php

```diff
@@ -18,6 +18,9 @@
 
 class ArgumentResolver implements ValueResolverInterface
 {
+    public const DATA_MAPPER_ERROR_MESSAGE = 'Request contents is not valid';
+    public const DATA_MAPPER_ERROR_CODE = Response::HTTP_UNPROCESSABLE_ENTITY;
```

Comment:
Potřebuji nějaké validace provádět i přímo v aplikaci, kde ArgumentResolver nepokryje všechny potřeby. A chci zachovat error message. Tak jsem to přesunul do constanty.